pkg/ext/extension_apiserver.go

```diff
@@ -191,9 +196,9 @@ func NewExtensionAPIServer(ctx context.Context, wranglerContext *wrangler.Contex
 
 	additionalSniProviders = append(additionalSniProviders, sniProvider)
 
-	if err := CreateOrUpdateService(wranglerContext.Core.Service(), opts.AppSelector); err != nil {
-		return nil, fmt.Errorf("failed to create or update APIService: %w", err)
-	}
+	//if err := CreateOrUpdateService(wranglerContext.Core.Service(), opts.AppSelector); err != nil {
+	//	return nil, fmt.Errorf("failed to create or update APIService: %w", err)
+	//}
```

Comment:
If we don't need this, then let's remove the code entirely (and assuming no one's using CreateOrUpdateService we can delete that code too.